change/office-ui-fabric-react-2020-01-10-14-23-07-jg-ts37.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
This should have been `minor`.

Reply:
@kenotron Should we do any validation with `yarn change` somehow? If API files change, that could be a strong hint on change type.